config/deploy.rb

```diff
@@ -51,4 +51,20 @@
   end
   before :restart, :resquepoolrestart
 
+  # load the workflow configs
+  desc "Load workflow configurations"
+  task :loadworkflows do
+    on roles(:web) do
+      execute :rake, 'curation_concerns:workflow:load'
+  end
+  before :restart, :loadworkflows
+
```

Comment:
Admin Sets and Workflows are new functionality in sufia 7.3. Personally I don't think it's quite mature enough to start using. But you do have to have defaults in place for the code to work. Workflow documentation is forthcoming; Jeremy just said he has it slated for next week (many at ldcx also asking for it).